src/mo_values/prim.ml

```diff
@@ -382,4 +382,12 @@ let prim trap =
   | "canister_version" ->
       fun _ v k -> as_unit v; k (Nat64 (Numerics.Nat64.of_int 42))
 
+  (* fake *)
+  | "setCandidLimits" ->
+      fun _ v k -> k unit
+  | "getCandidLimits" ->
+      fun _ v k -> k (Tup [
+        Nat64 Numerics.Nat64.zero; Nat64 Numerics.Nat64.zero;
+        Nat64 Numerics.Nat64.zero; Nat64 Numerics.Nat64.zero])
```

Comment:
I wonder what the fourth item is. And the types are `Nat64` (while the other returned values are `Nat32`). Probably not relevant because it is a fake function.

Reply:
Good catch - I forgot to adjust this...